### DISCUSSION TITLE
How to update docker ?

### DISCUSSION THREAD

**Discussion by Novosti1** [btcpayserver/btcpayserver#4379] [Technical Support] [answered]
root@btcpay-KVM:~/BTCPayServer/btcpayserver-docker# **btcpay-update.sh**
Already up to date.
Using default tag: latest
latest: Pulling from btcpayserver/docker-compose-generator
Digest: sha256:3a66d72869c71e032396bd4e0cf997533c2118467be1a4706dcd2f11beb3ac8e
Status: Image is up to date for btcpayserver/docker-compose-generator:latest
docker.io/btcpayserver/docker-compose-generator:latest

My current version: 
© BTCPay Server v1.7.0.0+def5095d7 (Altcoins) 

but 1.7.1 available already... 

what i am doing wrong ?

**Comment by dennisreimann** [comment, marked as answer]
Please try again, it is available now.
